src/vector/_methods.py

```diff
@@ -2969,16 +2969,16 @@ def to_Vector3D(
         if sum(x is not None for x in (z, theta, eta)) > 1:
             raise TypeError("Only one non-None parameter allowed")
 
-        z = 0
+        coord_value = 0.0
         l_type = LongitudinalZ
```

Comment:
You can set this to something like `l_type: type[Longitudinal] = LongitudinalZ` or something like that, IIRC, to avoid the type ignore.

Reply:
I did try this (in fact, IIRC, this was added by you in one of the commits above), but `mypy` still required the type ignore statements :(